src/__tests__/components/Ratings.test.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import renderer from 'react-test-renderer';
+import { Ratings, mapStateToProps, mapDispatchToProps } from '../../components/Rating/Ratings';
+import { ratingData } from '../../__mocks__/dummyData';
+import initialState from '../../redux/initialState.json';
+
+let wrapper;
+const props = {
+  loadingRatings: true,
+  ratings: [ratingData],
+  fetchRatings: jest.fn().mockImplementation(() => Promise.resolve({ status: 200 })),
+  match: {
+    params: {
+      articleSlug: 'article-slug',
+    },
+  },
+  history: { push: jest.fn() },
+};
+
+describe('<Ratings />', () => {
+  beforeEach(() => {
+    wrapper = mount(<Ratings {...props} />);
+  });
```

Comment:
@NkFab Can you add an empty line after this line for better readability?

Reply:
Done